src/main/java/se/liu/ida/hefquin/engine/wrappers/graphqlwrapper/data/impl/GraphQLEntrypointType.java

```diff
@@ -0,0 +1,7 @@
+package se.liu.ida.hefquin.engine.wrappers.graphqlwrapper.data.impl;
+
+public enum GraphQLEntrypointType {
```

Comment:
Add javadoc that explains the purpose of the enum and its different constant values